pkg/build/graph/edges.go

```diff
@@ -11,9 +13,10 @@ import (
 )
 
 const (
-	BuildInputImageEdgeKind = "BuildInputImage"
-	BuildOutputEdgeKind     = "BuildOutput"
-	BuildInputEdgeKind      = "BuildInput"
+	BuildTriggerImageEdgeKind = "BuildTriggerImage"
```

Comment:
I know it wasn't here before, but can you add GoDoc describing the head and tail node types of the edges and what they represent?